maps/randomvaults/objects.dm

```diff
@@ -198,6 +198,70 @@
 	..()
 	pod_code = "[rand(10000, 99999.0)]"
 
+/area/vault/amelab
+	name = "\improper Nanotrasen Experimental AME Lab"
+	requires_power = 1
+
+/area/vault/amelab/lab
+	requires_power = 0
+	jammed = 2
+	color = "blue"
+
+/obj/item/weapon/paper/amelab1
+	name = "paper- 'Initial Report'"
+	info = "I have arrived at the lab. The builders have just finished setting up the atmos and have departed. The station is using so little power that the P.A.C.M.A.N. in the maintenance is more than enough to recharge the APC. Centcomm should send the parts for the reactor by tomorrow, I can't wait to build it."
+
+/obj/item/weapon/paper/amelab2
+	name = "paper- 'Second Report'"
+	info = "The cargo shuttle dropped the parts about an hour ago and I've somehow already finished assembling the Antimatter Engine. In fact, the lab is running on its power right now. It's such a fool-proof system, you just pulse the parts with a multitool and it assembles on its own, not sure how anyone could mess it up really. I've shutdown the P.A.C.M.A.N., it might come in handy later in case of emergency. Antimatter is a much more efficient energy source, with just one jar this small lab could stay online for aeons."
+
+/obj/item/weapon/paper/amelab3
+	name = "paper- 'Third Report'"
+	info = "As instructed in Phase One of the experiment, I've started slowly raising the fuel injection. The engine seems to remain stable at up to 4 units of antimatter per injection, beyond that it quickly starts losing structural integrity and the injections become quite loud. I have determined that the highest safe amount of fuel units per injection is equal to twice the amount of cores."
+
+/obj/item/weapon/paper/amelab4
+	name = "paper- 'Fourth Report'"
+	info = "Centcomm has shipped me some much larger fuel jars in preparation for Phase Two. A scientist will be coming, an Anomalist specifically. I'm not quite versed into Exotic Particle science but they will test alternative ways to raise the power output. This can go wrong in so many ways but we're all well aware of the risks, and the pay is good enough. Worst case scenario the lab explodes and we'll be the only casualties."
+
+/obj/item/weapon/paper/amelab5
+	name = "paper- 'Fifth Report'"
+	info = "The anomalist has started their experiments on the engine, they brought various devices and machines I do not recognize. Regardless, their radiations their tools emit seem to have a positive effect on the power output. Looks like we're on the verge of a technological breakthrough! With the last deliveries we also ordered a portable SMES kit. If we can export enough surplus energy we might not only earn ourselves a bonus, but also prove the commercial applications of the enhanced AME."
```

Comment:
The "their radiations their tools emit" part feels off.

Reply:
Yep that's a typo

```suggestion
	info = "The anomalist has started their experiments on the engine, they brought various devices and machines I do not recognize. Regardless, the radiations their tools emit seem to have a positive effect on the power output. Looks like we're on the verge of a technological breakthrough! With the last deliveries we also ordered a portable SMES kit. If we can export enough surplus energy we might not only earn ourselves a bonus, but also prove the commercial applications of the enhanced AME."
```